apis/infix/atrium-api-infix/src/commonTest/kotlin/ch/tutteli/atrium/api/infix/en_GB/samples/IterableLikeToContainInAnyOrderCreatorSamples.kt

```diff
@@ -0,0 +1,167 @@
+import kotlin.test.Test
+
+class IterableLikeToContainInAnyOrderCreatorSamples {
+    @Test
+    fun value() {
+        expect(listOf("A", "B")) toContain o inAny order exactly 1 value "A"
+        expect(listOf("A", "B", "A", "B")) toContain o inAny order atLeast 2 value ("A")
+        expect(listOf("A", "B", "B")) toContain o inAny order atMost 2 value "B"
+
+        fails {
+            expect(listOf("A", "B")) toContain o inAny order exactly 2 value "A"
+            expect(listOf("A", "B", "A", "B")) toContain o inAny order atLeast 3 value "A"
+            expect(listOf("A", "B", "B", "B")) toContain o inAny order atMost 2 value "B"
+        }
+    }
+
+    @Test
+    fun values() {
+        expect(listOf("A", "B")) toContain o inAny order exactly 1 the values("B", "A")
+        expect(listOf("A", "B", "A", "B")) toContain o inAny order atLeast 2 the values("B", "A")
+        expect(listOf("A", "B", "B")) toContain o inAny order atMost 2 the values("B", "A")
+
+        fails {
+            expect(listOf("A", "B")) toContain o inAny order exactly 2 the values("B", "A")
+            expect(listOf("A", "B", "A", "B")) toContain o inAny order atLeast 3 the values("B", "A")
+            expect(listOf("A", "B", "B", "B")) toContain o inAny order atMost 2 the values("B", "A")
+        }
+    }
+
+    @Test
+    fun entry() {
+        expect(listOf("A", "B")) toContain o inAny order exactly 1 entry {
+            it toEqual "A"
+        }
+
+        expect(listOf("A", null, null)) toContain o inAny order exactly 2 entry null // null is identified
+
+        expect(listOf("A", "B", "A", "B")) toContain o inAny order atLeast 2 entry {
+            it toEqual "A"
+        }
+
+        expect(listOf("A", "B", "B")) toContain o inAny order atMost 2 entry {
+            it toEqual "A"
+        }
+
+        fails { // because the count of "A" is not 2
+            expect(listOf("A", "B")) toContain o inAny order exactly 2 entry {
+                it toEqual "A"
+            }
+        }
+
+        fails { // because all elements are not null
+            expect(listOf("A", "B", "C")) toContain o inAny order exactly 1 entry null
+        }
+
+        fails { // because assertionCreatorOrNull is non-null and has no expectation
+            expect(listOf("A", "B", "A", "B")) toContain o inAny order exactly 1 entry {
+                /* do nothing */
+            }
+        }
+
+        fails { // because the count of "A" is less than 3
+            expect(listOf("A", "B", "A", "B")) toContain o inAny order atLeast 3 entry {
+                it toEqual "A"
+            }
+        }
+
+        fails { // because the count of "B" is more than 2
+            expect(listOf("A", "B", "B", "B")) toContain o inAny order atMost 2 entry {
+                it toEqual "B"
+            }
+        }
+    }
+
+    @Test
+    fun entries() {
+        expect(listOf("A", "B")) toContain o inAny order exactly 1 the entries(
+            { toEqual("B") },
```

Comment:
```suggestion
            { it toEqual "B" },
```
also in other places further below

Reply:
Done ✅